java/templates/_helpers.tpl

```diff
@@ -89,15 +89,19 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- end -}}
 
 {{- define "java.tests.metadata" -}}
```

Comment:
version bump required? there could be released charts in the wild with this?

Reply:
Bumping to 2.14.0 ... but I am not sure there are charts released with "java.tests.metadata"

Reply:
i mean these are global namespaced, if someone imports another chart that  has the old version you could get that version of it the helper